qcengine/programs/psi4.py

```diff
@@ -53,36 +60,39 @@ def compute(self, input_model: 'ResultInput', config: 'JobConfig') -> 'Result':
         Runs Psi4 in API mode
         """
         self.found(raise_error=True)
-        import psi4
 
         # Setup the job
-        input_model = input_model.copy().dict()
-        input_model["nthreads"] = config.ncores
-        input_model["memory"] = int(config.memory * 1024 * 1024 * 1024 * 0.95)  # Memory in bytes
-        input_model["success"] = False
-        input_model["return_output"] = True
+        input_data = input_model.json_dict()
```

Comment:
like this. makes sure nothing unserializable lurking and bysteps the copy at the same time